string name			# For now this is a meaningfull name like 'left' or 'right'
geometry_msgs/Point position		# Center postion of MarkerBoard
geometry_msgs/Vector3 orientation	# Vector's orientation expressed in Rodrig.  cv::Rodrigues() converts this to rotation matrix R
